### PR TITLE
final version of getting started section

### PR DESCRIPTION
**Pull request by lilianabs**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -92,4 +95,4 @@
     <!-- instant.page -->
     <script src="//instant.page/1.0.0" type="module" integrity="sha384-6w2SekMzCkuMQ9sEbq0cLviD/yR2HfA/+ekmKiBnFlsoSvb/VmQFSi/umVShadQI"></script>
   </body>
-</html>
+</html>
```
Comment (DhairyaLGandhi) [inline _layouts/default.html:98]: You might want to fix the end of page changes between different editors to use the Linux ones

Comment (lilianabs) [reply _layouts/default.html:99]: I tried to fix it, please let me know if it didn't work.

File: _layouts/default.html [hunk 2]
```diff
@@ -95,5 +95,5 @@
     <!-- instant.page -->
     <script src="//instant.page/1.0.0" type="module" integrity="sha384-6w2SekMzCkuMQ9sEbq0cLviD/yR2HfA/+ekmKiBnFlsoSvb/VmQFSi/umVShadQI"></script>
   </body>
-  
-</html>
```
Comment (lilianabs) [inline _layouts/default.html]: Seems to be fixed :)

**Comment by DhairyaLGandhi** [comment]
@lilianabs you'd also need to show the training loop work. That is, define `ps` and structure `data` properly so if one were to copy and paste the code, it would work as expected. Also you might have to update the complete code section.

**Comment by lilianabs** [comment]
@DhairyaLGandhi thank you for your comments! I've modified the tutorial to include your suggestions.

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+<br>
+
+### Step 6: Train your model
+
+Training a model is the process of computing the gradients with respect to the parameters for each data point in the data. At every step, the optimiser updates all of the parameters until it finds a good value for them. In fact, you can write this process as a *for loop*. Notice that before training your model, you need to zip the training data as `data = zip(x, y)`. Also, you need to set `ps = params([W, b])` to indicate that you want the derivatives of `W` and `b`.
+
+
+You can execute the training process of your model as follows:
+
+```julia
```
Comment (DhairyaLGandhi) [inline getting_started.md:113]:
```suggestion
```julia
data = zip(x, y)
ps = params(W, b)
```

Comment (lilianabs) [reply getting_started.md:113]: Thanks for suggesting the change. It's updated :)

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+Flux.train!(loss, params(model), data, opt)
+```
+
+<br>
+
+where:
+* **loss** is the loss function that you defined in [Step 3](#step-3-define-a-loss-function).
+* **params(model)** are the trainable parameters of the model. It uses the [params Flux function](https://fluxml.ai/Flux.jl/stable/training/training/#Model-parameters-1) to track the parameters.
```
Comment (DhairyaLGandhi) [inline getting_started.md:142]:
```suggestion
* **params(model)** are the trainable parameters of the model. It uses the [`Flux.params` function](https://fluxml.ai/Flux.jl/stable/training/training/#Model-parameters-1) to track the parameters.
```

**Review by DhairyaLGandhi — COMMENTED** [review]
Looks good thanks!
